### PR TITLE
[new] Add bootstrap4-duallistbox with npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by olafcm**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/istvan-ujjmeszaros/bootstrap-duallistbox
* Official website (optional, not the repository): https://www.virtuosoft.eu/code/bootstrap-duallistbox/
* NPM package url (optional): https://www.npmjs.com/package/bootstrap4-duallistbox
* License and its reference: Apache-2.0
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 45
  * Count of stars: 490
  * Count of forks: 215
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [X] This lib was not found on cdnjs repo
* [X] Doesn't already exist / has duplicate issue and PR
* [X] The lib has notable popularity
  * [X] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [X] More than 800 downloads stats per month on npm registry
* [X] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [X] Has valid tags for each versions (for git auto-update)
* [X] Auto-update setup
* [X] Auto-update target/source is valid.
* [X] Auto-update filemap is correct.

## Git commit checklist

* [X] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [X] The parent of the commit(s) in the PR is not older than 3 days.
* [X] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by olafcm** [comment]
This pull request is almost ready for merge.
How to resolve this last issue: 
> /ajax/libs/bootstrap4-duallistbox/4.0.1/bootstrap-duallistbox.min.js does not exist but is referenced in package.json

**Comment by PeterDaveHello** [comment]
@olafcm removal of the verion info will resolve it

**Comment by olafcm** [comment]
@PeterDaveHello That worked perfect, thank you 😃

File: ajax/libs/bootstrap4-duallistbox/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "npmName": "bootstrap4-duallistbox",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "bootstrap-duallistbox.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap4-duallistbox/package.json]: This should be `jquery.bootstrap-duallistbox.min.js`, I'll fix it.

**Comment by olafcm** [comment]
Thank you, @PeterDaveHello, working perfect 😎

**Comment by PeterDaveHello** [comment]
@olafcm thank you

**Pull request merged**